app/src/src/components/views/LikedUserView.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+// import ChatBubbleIcon from "@material-ui/icons/ChatBubble";
```

Comment:
If this is not used, we can delete this import

Reply:
👍